src/components/Header.jsx

```diff
@@ -1,15 +1,68 @@
 import React from "react";
-import { Grid } from "semantic-ui-react";
+import { Grid, Button } from "semantic-ui-react";
+import { Link, useHistory } from "react-router-dom";
+import auth from "../modules/auth.js";
 
-const Header = () => {
-  return (
-    <Grid columns={3} divided centered>
-      <Grid.Row>
-        <h1 id="header">Daily News Sense</h1>
-      </Grid.Row>
+const date = new Date();
+const currentTime = date.getHours();
+
+let time;
 
+if (currentTime < 12) {
+  time = "Morning";
+} else if (currentTime < 18) {
+  time = "Afternoon";
+} else {
+  time = "Evening";
+}
+
+const Header = (props) => {
+  const history = useHistory();
+  const logout = async () => {
+    try {
+      await auth.signOut();
+      props.setAuthenticated(false);
+      history.push("/");
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  return (
+    <Grid>
       <Grid.Row>
-        <Grid.Column></Grid.Column>
+        <Grid.Column>
+          <h1 id="header" style={{ textAlign: "center" }}>
+            <span>D</span>aily <span>N</span>ews <span>S</span>ense
```

Comment:
Neat, but very space consuming on mobile, according to screenshot.

Reply:
it is the font size not the span, we will add to the responsiveness in Native I believe